nile-oe-init-build-env

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env
+
+SCRIPT_ROOT=$(realpath $(dirname ${BASH_SOURCE:-${0}}))
+
+
+usage() {
+	cat >&2 <<EOF
+$(basename ${BASH_SOURCE}) [-h] \
+	[[--] OE_INIT_ARGS...]
+
+Arguments:
+-h, --help    Print this usage text and exit.
+-o, --org
+        Enable configurations which are only valid when the build host is connected
+        to the NI corporate network. Community builders should not use this
+        setting.
+
+EOF
+}
+enable_ni_org_conf=false
+positionals=()
+
+set -u
+while [ $# -ge 1 ]; do case "$1" in
+	-h|--help)
```

Comment:
I see that `ni-oe-init-build-env` has other args like `download-dir` and `sstate-mirror`. Is the plan to add them in future as they're required?